src/main/java/org/gridsuite/study/server/service/NetworkService.java

```diff
@@ -73,6 +73,16 @@ public Network getNetwork(UUID networkUuid, PreloadingStrategy strategy, String
         }
     }
 
+    public boolean isNetworkExisting(UUID networkUuid) {
```

Comment:
Why don't you just call the `getNetwork` from `networkStoreService` directly ?
And you need to return false only if it's a 
`PowsyblException("Network '" + uuid + "' not found");`
Otherwise, let the exception spread

Reply:
```suggestion
    public boolean doesNetworkExist(UUID networkUuid) {
```
I think it's more correct

Reply:
> I think it's more correct

it's more correct in english, but I don't think we have any function called this way

> And you need to return false only if it's a
> PowsyblException("Network '" + uuid + "' not found");
> Otherwise, let the exception spread

I will change it but catch all "PowsyblException" as done is "getNetwork" in order to keep the same behaviour